packages/flutter/lib/src/material/slider.dart

```diff
@@ -661,7 +655,7 @@ class _RenderSlider extends RenderBox {
       _active = false;
       _currentDragValue = 0.0;
       _state.overlayController.reverse();
-      if (showValueIndicator && !_state.interactionTimer.isActive) {
+      if (showValueIndicator && _state.interactionTimer?.isActive != true) {
```

Comment:
...then here you can just check if it's null.

Reply:
OK.